apps/server/src/events/event-handlers/notify-event-comment-created.handler.ts

```diff
@@ -1,125 +1,93 @@
 import { injectableClass } from 'ditox';
+import { eq } from 'drizzle-orm';
 
-import { ConfigPort } from '../../infrastructure/config/config.port';
 import { EventHandler } from '../../infrastructure/cqs/event-handler';
-import { EmailRendererPort } from '../../infrastructure/email/email-renderer.port';
 import { TranslationPort } from '../../infrastructure/translation/translation.port';
-import { SubscriptionService } from '../../notifications/subscription.service';
-import { CommentRepository } from '../../persistence/repositories/comment/comment.repository';
-import { EventRepository } from '../../persistence/repositories/event/event.repository';
-import { MemberRepository } from '../../persistence/repositories/member/member.repository';
+import { GetNotificationContext, NotificationService } from '../../notifications/notification.service';
+import { Database } from '../../persistence/database';
+import * as schema from '../../persistence/schema';
 import { TOKENS } from '../../tokens';
 import { EventCommentCreated } from '../event-events';
 
+type Member = typeof schema.members.$inferSelect;
+type Event = typeof schema.events.$inferSelect;
+type Comment = typeof schema.comments.$inferSelect;
+
 export class NotifyEventCommentCreated implements EventHandler<EventCommentCreated> {
-  static inject = injectableClass(
-    this,
-    TOKENS.config,
-    TOKENS.translation,
-    TOKENS.memberRepository,
-    TOKENS.eventRepository,
-    TOKENS.commentRepository,
-    TOKENS.subscriptionService,
-    TOKENS.emailRenderer,
-  );
+  static inject = injectableClass(this, TOKENS.translation, TOKENS.database, TOKENS.notificationService);
 
   constructor(
-    private readonly config: ConfigPort,
     private readonly translation: TranslationPort,
-    private readonly memberRepository: MemberRepository,
-    private readonly eventRepository: EventRepository,
-    private readonly commentRepository: CommentRepository,
-    private readonly subscriptionService: SubscriptionService,
-    private readonly emailRenderer: EmailRendererPort,
+    private readonly database: Database,
+    private readonly notificationService: NotificationService,
   ) {}
 
-  private get appBaseUrl(): string {
-    return this.config.app.baseUrl;
-  }
-
-  async handle({ entityId: eventId, commentId }: EventCommentCreated): Promise<void> {
-    const event = await this.eventRepository.getEvent(eventId);
-    assert(event);
-
-    const organizer = await this.memberRepository.getMember(event.organizerId);
-    assert(organizer);
-
-    const comment = await this.commentRepository.getComment(commentId);
-    assert(comment);
+  async handle({ entityId: eventId, commentId, authorId }: EventCommentCreated): Promise<void> {
+    const event = defined(
+      await this.database.db.query.events.findFirst({
+        where: eq(schema.events.id, eventId),
+        with: {
+          organizer: true,
+          comments: true,
+          participants: true,
```

Comment:
**suggestion**

do not include participant with negative answer

Reply:
s/suggestion/isue/

Reply:
fc82172